cmd/tempo/app/modules.go

```diff
@@ -837,7 +861,7 @@ func (t *App) setupModuleManager() error {
 		Ingester:                      {Common, Store, MemberlistKV, PartitionRing},
 		MetricsGenerator:              {Common, OptionalStore, MemberlistKV, PartitionRing},
 		MetricsGeneratorNoLocalBlocks: {Common, GeneratorRingWatcher},
-		Querier:                       {Common, Store, IngesterRing, MetricsGeneratorRing, SecondaryIngesterRing},
+		Querier:                       {Common, Store, IngesterRing, MetricsGeneratorRing, SecondaryIngesterRing, PartitionRing},
```

Comment:
Is there any harm in having both the ingester/partition ring here? I suppose if we want dual checking read/write we will need it to be both available. Though we would need more code changes above, probably not a concern at the moment